### ISSUE TITLE
POST not working, all else works perfect

### ISSUE THREAD

**Issue by base64bits** [GoyaPtyLtd/RESTfm#19]
Thanks for making RESTfm open source and sharing it with the rest of the world, I love the tool, but have an issue:

I am running RESTfm in Filemaker Server 16 on a Mac Mini, I have been able to implement RESTfm for getting records, finding records and performing scripts, but I cannot use it for creating records, if I try to use the regular PHP XML I can create records just fine, but not with RESTfm.

Installation report page shows no errors and it saids RESTfm is working!

Here is my curl call:
curl -X POST \
  https://fms.domain.com/RESTfm/dataBaseName/layout/layoutName \
  -H 'Accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{"data":[{"name":"Jimi","lastName":"Hendrix"}]}'

Here is what I get back:
{
    "info": {
        "X-RESTfm-FM-Status": null,
        "X-RESTfm-FM-Reason": "",
        "X-RESTfm-Version": "5.0.0/20170821-1ee8f4e",
        "X-RESTfm-Protocol": "6",
        "X-RESTfm-Status": 500,
        "X-RESTfm-Reason": "FileMaker Error",
        "X-RESTfm-Method": "POST",
        "X-RESTfm-Trace": "exception 'RESTfm\\BackendFileMaker\\FileMakerResponseException' with message 'FileMaker Error' in /Library/FileMaker Server/HTTPServer/htdocs/RESTfm/lib/RESTfm/BackendFileMaker/FileMakerOpsRecord.php:88\nStack trace:\n#0 /Library/FileMaker Server/HTTPServer/htdocs/RESTfm/lib/RESTfm/OpsRecordAbstract.php(209): RESTfm\\BackendFileMaker\\FileMakerOpsRecord->_createRecord(Object(RESTfm\\Message\\Message), Object(RESTfm\\Message\\Record), 1)\n#1 /Library/FileMaker Server/HTTPServer/htdocs/RESTfm/lib/RESTfm/OpsRecordAbstract.php(167): RESTfm\\OpsRecordAbstract->createBulk(Object(RESTfm\\Message\\Message))\n#2 /Library/FileMaker Server/HTTPServer/htdocs/RESTfm/lib/uriLayout.php(283): RESTfm\\OpsRecordAbstract->createSingle(Object(RESTfm\\Message\\Message))\n#3 [internal function]: uriLayout->post(Object(RESTfm\\Request), 'dataBaseName', 'layoutName')\n#4 /Library/FileMaker Server/HTTPServer/htdocs/RESTfm/lib/tonic/lib/tonic.php(695): call_user_func_array(Array, Array)\n#5 /Library/FileMaker Server/HTTPServer/htdocs/RESTfm/lib/RESTfm/Resource.php(54): Tonic\\Resource->exec(Object(RESTfm\\Request))\n#6 /Library/FileMaker Server/HTTPServer/htdocs/RESTfm/RESTfm.php(99): RESTfm\\Resource->exec(Object(RESTfm\\Request))\n#7 {main}",
        "X-RESTfm-Profile": "122ms 2.2M 128M"
    }
}

I just tried the RESTfm version before the last one just to test, but the result is the same.

**Comment by nickorr** [comment]
base64bits,

GitHub is for plugin code compiling issues, not general FileMaker usage support.  For support please refer here : https://baseelementsplugin.zendesk.com/hc/en-us/articles/115001543347-What-is-plugin-sponsorship- or consider some of the free forums such as the FileMaker community.

Cheers,
Nick

**Comment by base64bits** [comment]
Hi Nick, thanks for the time you took to reply, I have tried the FileMaker support forums and none seems to know how to fix this issue, while I am new to Github I though this was a bug in the code or some compatibility issue with my setup. I have invested some time in trying to get this to work, so far for all other usage RESTfm has been awesome, I just cannot create records, I guess if I do not find a solution I will resource to using the FileMake PHP and XML API to create records.

**Comment by base64bits** [comment]
After looking for help in other places and not getting any results I started to do some more troubleshooting, just now I got Jimi Hendrix into my FileMaker system, the solution was to uninstall FileMaker server and reinstall it again following all the steps you would normally follow when upgrading etc. Backup schedules, backup databases, backup plugins, stop the FileMaker services, uninstall, restart, install and restore data and schedules, there are more steps involved but the solution was to basically uninstall and install the FileMaker server software again.
 
Hope this helps someone if they have the same problem.
